<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainWidget</string>
  </property>
  <widget class="QLabel" name="label_x">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>530</y>
     <width>99</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_y">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>570</y>
     <width>99</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_x">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>530</y>
     <width>55</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>14</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_y">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>570</y>
     <width>55</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_black">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>610</y>
     <width>135</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>黑棋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_white">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>670</y>
     <width>135</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>白棋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_restart">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>730</y>
     <width>135</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>重新开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>135</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="btn_whitefirst">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>430</y>
     <width>122</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>白棋优先</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_conn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>380</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查找玩家</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
